### DISCUSSION TITLE
Ability to import/export all collections as template

### DISCUSSION THREAD

**Discussion by maartenvn** [directus/directus#3042] [Feature Requests]
I would be really handy to export/import all collections (only the models, not the data) into a JSON document that is independent of Directus versions. This would make migrating a lot easier between different versions of Directus.

It would also allow for easy migration from a Development instance to a Production instance (running on different database software, for example)

**Comment by rijkvanzanten** [comment]
Yes! This is an in works ticket 🙂 

https://github.com/directus/next/issues/333

**Reply by pwFoo** [reply]
Hi, link destination ist a 404. Issued moved?

**Reply by rijkvanzanten** [reply]
There ya go https://github.com/directus/directus/issues/2927

**Reply by rijkvanzanten** [reply]
There ya go https://github.com/directus/directus/issues/2927

**Reply by pwFoo** [reply]
Thanks! Would be a very convenient feature to backup / deploy database structure and directus settings.

**Comment by rijkvanzanten** [comment]
This now exists in the form of the schema snapshot and apply endpoints 🙂
